src/ncp/ncp_openthread.cpp

```diff
@@ -209,6 +256,15 @@ otError ControllerOpenThread::ApplyFeatureFlagList(const FeatureFlagList &aFeatu
     otNat64SetEnabled(mInstance, aFeatureFlagList.enable_nat64());
 #endif
 
+    if (aFeatureFlagList.enable_detailed_logging())
+    {
+        error = SetOtbrAndOtLogLevel(ConvertProtoToOtbrLogLevel(aFeatureFlagList.detailed_logging_level()));
+    }
+    else
+    {
+        error = SetOtbrAndOtLogLevel(otbrLogGetDefaultLevel());
```

Comment:
If `OT_LOG_LEVEL` is explicitly set to some value (on a product build), here we may change it at run-time to some other value?

Reply:
It looks like otbr doesn't use OT_LOG_LEVEL, but starts from OTBR_LOG_INFO, then takes in otbr command line argument `-d`, and set that value to otbr and ot log level.
Here I'm trying to set back to the above value.

Reply:
Got it. Make sense.

I assumed that the `-d` was specifying `otbr` log level only.
- Looking at the functions in `common/logging.hpp`, they all seem to be dealing with logging within `otbr` itself.
- So I thought that the logging in `otbr` and OT core were treated differently and may even use two different levels.

But looks like in `ControllerOpenThread::Init(void)` we call `otLoggingSetLevel()` and set the OT log level to the same (converted) level:
```
void ControllerOpenThread::Init(void)
{
    otbrError  error = OTBR_ERROR_NONE;
    otLogLevel level = ConvertToOtLogLevel(otbrLogGetLevel());

    VerifyOrExit(otLoggingSetLevel(level) == OT_ERROR_NONE, error = OTBR_ERROR_OPENTHREAD);
```